src/docs/blueprint_documentation.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Packaging Theia as a Desktop Product
+---
+
+# Packaging Theia as a Desktop Product
+
+Eclipse Theia Blueprint product is a template to showcase the capabilities of Theia as well as how to build desktop-based products based on the platform.
+Theia Blueprint assembles a selected subset of existing Theia features and extensions.
+We provide installers for Theia Blueprint to be downloaded (see below) from the Theia webpage.
+In the respective git Repository you can also find the source code the product and its installers were built from.
+This is also the starting point for this documentation that is about how to customize this template to build your own custom Theia-based product and installers and packaging for installing the Desktop-based version of your custom product on all major operating systems.
+
+This documentation will provide you with a basic introduction how the Blueprint product and its installers/packaging are built.
+We then detail how to customize the product by adjusting the built-in VS Code and Theia extensions for the product and the branding of the product.
+Finally, we conclude with how to publish and update your products and their packaging and installers.
```

Comment:
I'm not sure we need to describe the sections, this paragraph will become outdated quickly as we include new sections. If we'd like to outline the later sections, we can simply use an bullet outline.

Reply:
Removed the paragraph